src/components/Subheader.js

```diff
@@ -3,14 +3,9 @@ import { Row, Col } from "react-bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
 import { getSubheaderList } from "../functions/getSubheaderList";
 
-export function Subheader(props) {
+export const Subheader = props => {
```

Comment:
please deconstruct always all `props`

Reply:
Props deconstructed in Subheader and List

Reply:
please always respond with a git commit hash (if possible), eg

d7c6b4faccadc17f0374f46cad795dab84c95557

so we know what you did

also, it's not about you to resolve a conversation, but for the person who requested the review

:+1: